src/project-3.js

```diff
@@ -87,6 +126,11 @@ const addCalculateDiscountPriceMethod = (storeItem) => {
   // discountPrice = 20 - (20 * .2)
   // Make sure you return storeItem after adding the method to it
   // hint: arrow functions don't bind a this
+  // storeItem.calculateDiscountPrice = function () {
```

Comment:
You'd need to name your function here. After function keyword, adds a name to it.

Reply:
thanks, I will work on this again later after lectures are over.

Reply:
Did you get this figured out?